pkg/constants/constants.go

```diff
@@ -105,7 +111,45 @@ const (
 	EventsQueryTag   = "Events Query"
 	AuditingQueryTag = "Auditing Query"
 
-	AlertingTag = "Alerting"
+	AlertingTag          = "Alerting"
+	MsgLen               = 512
+	HelmRepoSyncStateLen = 10
+	HelmAuditLen         = 10
+
+	//app version state
+	StateDraft     = "draft"
+	StateSubmitted = "submitted"
+	StatePassed    = "passed"
+	StateRejected  = "rejected"
+	StateSuspended = "suspended"
+	StateActive    = "active"
+
+	RepoStateSuccessful = "successful"
+	RepoStateFailed     = "failed"
+	RepoStateSyncing    = "syncing"
+
+	HelmStatusActive      = "active"
+	HelmStatusCreating    = "creating"
+	HelmStatusDeleting    = "deleting"
+	HelmStatusUpgrading   = "upgrading"
+	HelmStatusRollbacking = "rollbacking"
+	HelmStatusPending     = "pending"
+	HelmStatusSuccessful  = "successful"
+	HelmStatusFailed      = "failed"
+
+	AttachmentTypeScreenshot = "screenshot"
+	AttachmentTypeIcon       = "icon"
+
+	HelmApplicationAppStoreSuffix  = "-store"
+	HelmApplicationIdPrefix        = "app-"
+	HelmRepoIdPrefix               = "repo-"
+	HelmApplicationVersionIdPrefix = "appv-"
+	HelmCategoryIdPrefix           = "ctg-"
+	HelmAttachmentPrefix           = "att-"
+	HelmReleasePrefix              = "rls-"
+	UncategorizedName              = "uncategorized"
+	UncategorizedId                = "ctg-uncategorized"
+	AppStoreRepoId                 = "repo-helm"
```

Comment:
I think these constants can be moved to `/pkg/api/application/v1alpha1`.

Reply:
done